packages/plugins/i18n/server/src/controllers/validate-locale-creation.ts

```diff
@@ -23,7 +23,7 @@ const validateLocaleCreation: Common.MiddlewareHandler = async (ctx, next) => {
     return next();
   }
 
-  const locale = get('plugins.i18n.locale', query);
+  const locale = get('locale', query);
   const relatedEntityId = get('plugins.i18n.relatedEntityId', query);
```

Comment:
do we keep the relatedEntityId inside the plugins.i18n param too? 
Maybe we can do that in another pr